src/main/java/com/jcabi/http/request/ApacheRequest.java

```diff
@@ -124,9 +137,11 @@ public String getMethod() {
             final URI uri = URI.create(home);
             req.setConfig(
                 RequestConfig.custom()
-                    .setCircularRedirectsAllowed(false)
-                    .setRedirectsEnabled(false)
-                    .build()
+                        .setCircularRedirectsAllowed(false)
```

Comment:
@benhinssen can't we move these lines back a bit? The old alignment was correct.